src/app/[lng]/(common)/notice/[id]/AdditionalNotices.tsx

```diff
@@ -14,11 +13,12 @@ const AdditionalNotices = async ({
   notice,
   additionalContents,
   t,
-  lng,
 }: AdditionalNoticesProps & PropsWithLng) => {
   return (
-    <div className={'flex flex-col gap-4'}>
+    <div className={'flex flex-col gap-[18px]'}>
       {additionalContents.map((content, index) => {
+        const timeAgo = dayjs(content.createdAt).fromNow();
+
         const lastDeadline =
           index > 0 ? additionalContents[index - 1].deadline : notice.deadline;
```

Comment:
이거는 이번 PR에서 나온건 아니지만..
```suggestion
          additionalContents[index - 1]?.deadline ?? notice.deadline;
```

이건 어때요?